lte/gateway/c/core/oai/tasks/nas5g/src/ies/M5GUESecurityCapability.cpp

```diff
@@ -98,7 +98,11 @@ int UESecurityCapabilityMsg::DecodeUESecurityCapabilityMsg(
     ue_sec_capability->eia7 = *(buffer + decoded) & 0x1;
     decoded++;
 
-    // decoded = type_len + length_len + ue_sec_capability->length;
+    // Skipping the remaining bytes as not supported
+    if (ue_sec_capability->length > (decoded - (type_len + length_len))) {
+      decoded +=
+          ue_sec_capability->length - (decoded - (type_len + length_len));
+    }
```

Comment:
the fix is needed to encode function also and verify encode and decode functions of  UE security capabilities

Reply:
Addressed Review Comment